src/librustc_mir/transform/instcombine.rs

```diff
@@ -106,6 +128,18 @@ impl Visitor<'tcx> for OptimizationFinder<'b, 'tcx> {
             }
         }
 
+        // find Ne(_place, false) or Ne(false, _place)
+        if let Rvalue::BinaryOp(BinOp::Ne, l, r) = rvalue {
+            // is left constant false and right place?
```

Comment:
Why not just show the representation? The next feels hard to comprehend.
```suggestion
            // (false, _place)
```

Reply:
Good point, fixed